e2e/LongHaul/device/IotHubConnectionStringHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.IoT.Thief.Device
+{
+    internal class IotHubConnectionStringHelper
+    {
+        public string HostName { get; private set; }
+
+        public string DeviceId { get; private set; }
+
+        public void GetValuesFromConnectionString(string connectionString)
```

Comment:
I suggest making this static and have it return an instance, or perhaps better, turn it into a ctor.